test/scripts/e2e-kind.sh

```diff
@@ -42,6 +47,17 @@ should resolve connection reset issue #74839
 # TO BE FIXED BY https://github.com/kubernetes/kubernetes/pull/129049
 Services should be able to switch session affinity for NodePort service
 
+# TO BE FIXED BY https://github.com/kubernetes/kubernetes/issues/130434
```

Comment:
these 4 conformance skip tests might go into a separate commit and it helps to revert after this PR is merged.